Simplenote/SeparatorsView.swift

```diff
@@ -0,0 +1,131 @@
+import Foundation
+
+open class SeparatorsView: UIView {
+    // MARK: - Public Properties
+    open var leftVisible = false {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+    open var leftColor = UIColor.simplenoteDividerColor {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+     open var leftWidthInPoints = CGFloat(3) {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+     open var topVisible = false {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+    open var topColor = UIColor.simplenoteDividerColor {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+    open var topHeightInPixels = CGFloat(1) {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+    open var topInsets = UIEdgeInsets.zero {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+    open var bottomVisible = false {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+    open var bottomColor = UIColor.simplenoteDividerColor {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+    open var bottomHeightInPixels = CGFloat(1) {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+    open var bottomInsets = UIEdgeInsets.zero {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+    open override var frame: CGRect {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+
+
+
+    // MARK: - UIView methods
+    convenience init() {
+        self.init(frame: CGRect.zero)
+    }
+
+    required override public init(frame: CGRect) {
+        super.init(frame: frame)
+        setupView()
+    }
+    required public init(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)!
+        setupView()
+    }
+
+    open override func draw(_ rect: CGRect) {
+        super.draw(rect)
+
+        let scale = UIScreen.main.scale
+        let ctx = UIGraphicsGetCurrentContext()
+        ctx!.clear(rect)
```

Comment:
I know this was part of the original code (and I have NO IDEA why there's an IUO there).

Can we please `guard let else`, and drop the force unwraps?